src/controllers/users/matchUserContent.ts

```diff
@@ -1,16 +1,21 @@
 import { Request, Response, NextFunction } from 'express';
+import { CustomError } from '../../helpers';
 import { matchUserContent } from '../../services';
 
 export default async (req: Request, res: Response, next: NextFunction): Promise<void> => {
   const { body } = req;
-  const modifiedBody = {
-    ...body,
-    filmingCosts: body.filmingCosts.toString(),
-    advance: body.advance.toString(),
-    feePaid: body.feePaid.toString(),
-  };
 
   try {
+    if (body.recoveredCosts > (body.filmingCosts + body.advance + body.feePaid)) throw new CustomError("Recovered costs can't be greater than costs", 400);
```

Comment:
```suggestion
const remainingCost = body.filmingCosts + body.advance + body.feePaid
    if (body.recoveredCosts > remainingCost ) throw new CustomError("Recovered costs can't be greater than costs", 400);
```
if those accumalted values aren't `remaingCost` and instead are `accumulatedCost` or `totalCost` then pick whatever you see is more accurate and rename it